src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/AuthorizationServer/ApiAccess/form.html.twig

```diff
@@ -25,7 +25,7 @@
 
 {% extends '@PrestaShop/Admin/layout.html.twig' %}
 
-{% form_theme apiAccessForm '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
+{% form_theme apiAccessForm '@PrestaShop/Admin/Configure/AdvancedParameters/AuthorizationServer/ApiAccess/form_theme.html.twig' %}
```

Comment:
Optional but you use the `form_theme` option instead and set it in `ApiAccessType` via the `configureOptions` and `setDefaults` methods:
https://github.com/PrestaShop/PrestaShop/blob/030a383b64a5411275de02bb3b2708dfe5a8dc21/src/PrestaShopBundle/Form/Admin/Sell/Product/Combination/CombinationFormType.php#L116